### DISCUSSION TITLE
configuration

### DISCUSSION THREAD

**Discussion by giscus** [Yiov/vitepress-doc#20] [General]
# configuration

我的vitpress文档教程

http://localhost:5174/configuration.html

**Comment by MayRay822** [comment]
1111

**Reply by MayRay822** [reply]
123123123213123

**Comment by Moeus** [comment]


**Reply by Yiov** [reply]
如果本地可以正常显示，bulid就不可以，那就是你[base](https://vitepress.yiov.top/assets.html#base)没有正确配置

**Reply by Yiov** [reply]
这里直接用/即可，不需要加public，这个文件夹默认为根目录

```
head: [
  ['link', { rel: 'icon', href: '/logo.jpg' }]],

```

**Reply by Yiov** [reply]
这个官方没有提供更改，但是你可以通过F12审查元素中找到此css类，适当的调整，需要一点点知识，如果你不懂可以先搭建完成后，有一定了解了回头再来改

**Comment by shiliuboom** [comment]
构建纯净链接具体作用是啥

**Reply by Yiov** [reply]
就简洁干净吧，虽说有可能影响seo，但我也没有实测过是否真的影响，所以看个人喜好即可。

**Reply by shiliuboom** [reply]
有交流群吗，哥
